package.json

```diff
@@ -8,6 +8,7 @@
     "files": [
       "test/**/*.test.js"
     ],
+    "failFast": true,
     "nodeArguments": [
       "--loader=testdouble",
```

Comment:
once we drop node v18 support, we'll want to remember to remove this from this list